### PR TITLE
Fix: Use config instead of os.GetEnv.

### PR DESCRIPTION
**Pull request by eloycoto**
That ENV variable was not documented at all, and should use Config
structure.

Signed-off-by: Eloy Coto <eloy.coto@acalustra.com>

### CODE REVIEW

**Review by ydayagi — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -126,6 +126,10 @@ var Config struct {
 
 	// AutoApprovalprocess enable auto approval on devices
 	AutoApproval bool `envconfig:"AUTO_APPROVAL_PROCESS" default:"true"`
+
+	// If Webhooks are enabled, an admission webhook is created and checked when
+	// any user submits any change to any project-flotta.io CRD.
+	EnableWebhooks bool `envconfig:"ENABLE_WEBHOOKS" default:"false"`
```
[the remark this answers is not in the document]
Comment (eloycoto) [reply main.go:132]: Are you sure that was true by default? Should be true?

https://github.com/project-flotta/flotta-operator/pull/215/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L248

https://github.com/project-flotta/flotta-operator/blob/98bfe5f11d890725393c0cc8588f96ca1dc11dd3/main.go#L248-L252

[the remark this answers is not in the document]
Comment (masayag) [reply main.go:132]: +1 for true by default.
It should be a conscious decision to disable validation (and a good reason for it)

[the remark this answers is not in the document]
Comment (ydayagi) [reply main.go]: it was only disabled when u execute 'make run'

[the remark this answers is not in the document]
Comment (eloycoto) [reply main.go]: Updated to default to true.

**Review by masayag — APPROVED** [review]

**Pull request merged**